### ISSUE TITLE
Time limit 0 

### ISSUE THREAD

**Issue by dameert** [ems-project/EMSCoreBundle#274]
https://github.com/ems-project/EMSCoreBundle/blob/f48629242dc2adeb0892539cab90b0e4491de3c7/Controller/ContentManagement/JobController.php#L134

Set time limit 0 Will not override php-fpm time limitations! And platform specific limitations imposed by the hosting environment

**Comment by laurensmertens** [comment]
@dameert So it's just a matter of removing this line and configuring the time limit on the platform in question?

**Comment by dameert** [comment]
Nope, because then php would not allow more than 30s runtime. This should be solved using a message broker that uses a real command line. Which has no time limit by default.
